swarm/tools/runs_gc.py

```
@@ -75,12 +76,17 @@ def get_dir_size(path: Path) -> int:
-                    pass
+        # PERFORMANCE OPTIMIZATION (Bolt): Using os.scandir instead of path.rglob("*")
+        # for significantly better performance when calculating directory size.
+        with os.scandir(path) as it:
+            for entry in it:
+                if entry.is_dir(follow_symlinks=False):
+                    total += get_dir_size(Path(entry.path))
+                elif entry.is_file():
+                    try:
+                        total += entry.stat().st_size
+                    except OSError:
+                        pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)